### PR TITLE
api: log unsupported models

### PR DESCRIPTION
**Pull request by EmilienM**
It can be really useful to provide logs when a device (PF or VF) is not
supported and give the vendorId + deviceId; so the deployer can check
what device is being selected and compare with the ConfigMap which
describes the supported nics.

### CODE REVIEW

File: api/v1/helper.go [hunk 1]
```diff
@@ -103,6 +103,7 @@ func IsSupportedModel(vendorId, deviceId string) bool {
 			return true
 		}
 	}
+	log.Info("IsSupportedModel():", "Unsupported model:", "vendorId:", vendorId, "deviceId:", deviceId)
```
Comment (e0ne) [inline api/v1/helper.go:106]: IMO, it's better to use log.Warning here and below to get more attention into these log messages

**Review by e0ne — CHANGES_REQUESTED** [review]

File: api/v1/helper.go (hunk 1 above)
Comment (EmilienM) [reply api/v1/helper.go]: I agree but if you look at this file, all "errors" are reported via `log.Info`. The imported logger can do Info or Error. I could create another logger with glog and call `glog.Warning`, but... Up to you at this point.

Comment (EmilienM) [reply api/v1/helper.go:106]: @e0ne any thoughts?

File: Makefile [hunk 2]
```diff
@@ -11,11 +11,11 @@ export OPERATOR_EXEC?=oc
 BUILD_GOPATH=$(TARGET_DIR):$(TARGET_DIR)/vendor:$(CURPATH)/cmd
 IMAGE_BUILDER?=docker
 IMAGE_BUILD_OPTS?=
-DOCKERFILE?=Dockerfile
+DOCKERFILE?=Dockerfile.webhook
```
Comment (adrianchiris) [inline Makefile:14]: is this related to the PR ?

Comment (EmilienM) [reply Makefile:14]: sorry it was a mistake on my side, I'll drop that

**Comment by EmilienM** [comment]
@adrianchiris could I get this one approved please?

**Review by adrianchiris — APPROVED** [review]

**Comment by adrianchiris** [comment]
merging as i see @pliurh LGTM'd it
